NOM_visualizations/python/nom_data.ipynb

```diff
@@ -0,0 +1,3353 @@
+{
```

Comment:
<p>Wow great. I didn't realize you split these out. I am wondering if we should split them out even more so that the taxonomic python notebook can import them too. I think that's work for another time though. Just writing my thought here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>